docs/user_manual/components.md

```diff
@@ -135,12 +135,12 @@ if any of the faults in any of the scenarios within a batch are not three-phase
 
 #### Electric Model
 
-`line` is described by a π model, where 
+`line` is described by a $\pi$ model, where 
 
 $$
    \begin{eqnarray}
-      Z_\mathrm{series}    & = & r + \mathrm{j}x \\
-      Y_\mathrm{shunt} & = & 2 \pi fc/(tan \sigma +\mathrm{j})
+      & Z_{\text{series}} = r + \mathrm{j}x \\
+      & Y_{\text{shunt}} = \frac{2 \pi fc}{\tan \sigma +\mathrm{j}}
```

Comment:
in the table, it says `\tan \delta` but here it says `\tan \sigma`. Which of the two should it be?